f600233d1b154d507b782f84604bcb12/author_elective_update/sys_dictionary_x_g_dis_atat_acquisition_package_gfe_overview.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update>
+    <sys_dictionary action="DELETE" element="gfe_overview" table="x_g_dis_atat_acquisition_package">
```

Comment:
Deleting the `GFE Overview` table first, appears to also remove the `gfe_overview` column from the `Acq. Package`.